test/integration/test/philips_authentication_test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+
+var Prepper = require('../lib/make_suite.js');
+
+Prepper.makeSuite('Test Hue Authentication',function(){
+
+  var getterPayload = [{'feature': 'light/is-on'}];
+  var lights;
+
+  before ('turn on simulators', function (done){
+    Prepper.turnOnHueNupnp();
+    Prepper.turnOnHue(true);
+    Prepper.turnOnFoxbox();
+    Prepper.foxboxManager.foxboxLogin();
+    setTimeout(done,5000);
+  });
+
+  it('Send light query without authentication',function(){
+  
+    // collect all getters for the lightbulbs
+    return chakram.put(Prepper.foxboxManager.getterURL,getterPayload)
+    .then(function(listResponse) {
+      lights = Object.keys(listResponse.body);
+      expect(lights.length).equals(0);
+    });
+  });
+
+  describe('Authenticate with Philips Hue', function () {
+    before('Press the button', function(done) {
+      Prepper.philipshue_server.pressButton();
+      setTimeout(done,5000); //give time to collect light info
```

Comment:
I don't see anything wrong in this snippet. Maybe the before part contains an error? It should be close to
```js 
before('Press the button', function() {
  return Prepper.philipshue_server.pressButton();
})
```